game/src/components/TicTacToe.vue

```diff
@@ -0,0 +1,7 @@
+<!--Game-->
+<template>
+  <v-card variant="tonal"
+    ><v-card-title>Tic Tac Toe</v-card-title><v-card-text>placeholder</v-card-text
+    ><v-card-actions><v-btn to="/">Home</v-btn></v-card-actions></v-card
+  >
```

Comment:
Make sure you are using appropriate formatting for your code... E.g.
```suggestion
  <v-card variant="tonal">
    <v-card-title> Tic Tac Toe </v-card-title>
    <v-card-text> placeholder </v-card-text>
    <v-card-actions> 
      <v-btn to="/"> Home </v-btn> 
    </v-card-actions> 
</v-card>
```

You can also use `npm run lint` to auto format your code!